webapp/src/pages/ResponseList/components/ResponseListComponent.js

```diff
@@ -0,0 +1,494 @@
+import React, { Component } from 'react';
+import '../ResponseList.css';
+import { Trans, withTranslation } from 'react-i18next'
+import { responsesService } from '../../../services/responses/responses.service';
+import ReactTable from 'react-table';
+import "react-table/react-table.css";
+import { NavLink } from "react-router-dom";
+import { tagsService } from '../../../services/tags/tags.service';
+import FormTextBox from "../../../components/form/FormTextBox";
+import { reviewService } from '../../../services/reviews/review.service';
+import { ConfirmModal } from "react-bootstrap4-modal";
+import TagSelectorDialog from '../../../components/TagSelectorDialog';
+import FormSelect from "../../../components/form/FormSelect";
+import { createColClassName } from "../../../utils/styling/styling";
+
+class ResponseListForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            btnUpdate: false,
+            includeUnsubmitted: false,
+            responses: [],
+            filteredResponses: [],
+            tags: [],
+            filteredTags: [],
+            tagSelectorVisible: false,
+            selectedResponse: null,
+            error: null,
+            loading: true,
+            nameSearch : "",
+            tagSearch : "",
+            emailSearch: "",
+            gettingResponseList: false
+        }
+    }
+
+    componentDidMount() {
+        this.setState({ loading: true }, () => this.getResponseList());
+
+    }
+
+    getResponseList() {
+        this.setState({ gettingResponseList: true });
+        Promise.all([
+            tagsService.getTagList(this.props.event.id),
+            responsesService.getResponseList(this.props.event.id, this.state.includeUnsubmitted, []),
+        ]).then(([tagsResponse, responsesResponse]) => {
+            this.setState({
+                tags: tagsResponse.tags,
+                responses: responsesResponse.responses,
+                filteredResponses : responsesResponse.responses,
+                error: tagsResponse.error || responsesResponse.error,
+                loading: false,
+                gettingResponseList: false
+            });
+        });
+    }
+
+    refreshResponses() {
+        const { includeUnsubmitted } = this.state;
+
+        responsesService.getResponseList(this.props.event.id, includeUnsubmitted, []).then(resp => {
+            this.setState({
+                responses: resp.responses,
+                error: resp.error
+            });
+        });
+    }
+
+    generateColumns() {
```

Comment:
Looks like you're not using this method because it's all inside the render anyway, so can be deleted